src/libs/actions/Policy/Travel.ts

```diff
@@ -0,0 +1,24 @@
+import Onyx from 'react-native-onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+// TODO: Implement this when we have the Travel enable API
+function enablePolicyTravel(policyID: string, enabled: boolean) {
```

Comment:
FYI the name of the API is `EnablePolicyTravel`. `public static function enablePolicyTravel(string $authToken, string $policyID, bool $enabled): void`